tutorials/scripting/gdscript/static_typing.rst

```diff
@@ -279,6 +279,11 @@ get full autocompletion on the player variable thanks to that cast.
 
         player.damage()
 
+You can also simplify the expression using the ``is not`` operator::
+
+    if body is not PlayerController:
+        push_error("Bug: body is not PlayerController")
```

Comment:
```suggestion
    You can also simplify the expression using the ``is not`` operator::
    
        if body is not PlayerController:
            push_error("Bug: body is not PlayerController")
```
This needs to be indented one level deeper, so it appears in the note block, since it is conceptually related to the earlier example.

You should also relocate this new `is not` example to after the initial `is` example and before the `assert()` example. Note that you'll have to adjust the existing `or ``assert()`` statement::` line to make sense.